keyboards/handwired/nortontechpad/info.json

```diff
@@ -0,0 +1,52 @@
+{
+    "keyboard_name": "NortonTechPad",
+    "manufacturer": "NortonTech",
+    "url": "",
+    "maintainer": "NortonTech",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x9801",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": false,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D7", "E6", "B4", "B5"],
+        "rows": ["F7", "B1", "B3", "B2", "B6"]
+    },
+    "diode_direction": "COL2ROW",
+    "development_board": "promicro",
+    "layouts": {
+        "LAYOUT_numpad_5x4": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0, "y": 0},
+                {"matrix": [0, 1], "x": 1, "y": 0},
+                {"matrix": [0, 2], "x": 2, "y": 0},
+                {"matrix": [0, 3], "x": 3, "y": 0},
+
+                {"matrix": [1, 0], "x": 0, "y": 1},
+                {"matrix": [1, 1], "x": 1, "y": 1},
+                {"matrix": [1, 2], "x": 2, "y": 1},
+                {"matrix": [1, 3], "x": 3, "y": 2, "h": 2},
+
+                {"matrix": [2, 0], "x": 0, "y": 2},
+                {"matrix": [2, 1], "x": 1, "y": 2},
+                {"matrix": [2, 2], "x": 2, "y": 2},
+
```

Comment:
```suggestion

                {"matrix": [2, 0], "x": 0, "y": 2},
                {"matrix": [2, 1], "x": 1, "y": 2},
                {"matrix": [2, 2], "x": 2, "y": 2},
                {"matrix": [1, 3], "x": 3, "y": 2, "h": 2},

```

Reply:
This keyboard does not use the community layout, the matrix/keymap will not work with this change

Reply:
Thanks for the tip, I'll try this and see if works

For reference, the way I have the key layer out is the tall key is wired to col 3 and row 1, based on this do you think I should implement the changes Zvecr suggested?

Reply:
> I can't add on to zvecr's suggestion, but you should put the tall key on "y": 1

Updated suggestion to reflect this.